addons/l10n_jp_ubl_pint/__manifest__.py

```diff
@@ -1,6 +1,5 @@
 {
     'name': 'Japan - UBL PINT',
-    'countries': ['jp'],
```

Comment:
@william-andre I am confused about this module because it does not depend on `l10n_jp`

Reply:
It doesn't have to if it doesn't rely on any data/fields from that module